From 9a6ef5bf47738e35b97f60e935d478b202f85945 Mon Sep 17 00:00:00 2001
From: ElXreno <elxreno@gmail.com>
Date: Wed, 22 Jul 2020 19:57:27 +0300
Subject: [PATCH] rpm: drop not required sections from install

Signed-off-by: ElXreno <elxreno@gmail.com>
---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index e78897b..93b7733 100644
--- a/Makefile
+++ b/Makefile
@@ -16,9 +16,6 @@ base:
 	install -d $(DESTDIR)$(SBINDIR)
 	install -m0755 memavaild $(DESTDIR)$(SBINDIR)/memavaild
 
-	install -d $(DESTDIR)$(DOCDIR)
-	install -m0644 README.md $(DESTDIR)$(DOCDIR)/README.md
-
 	install -d $(DESTDIR)$(SYSCONFDIR)
 
 	sed "s|:TARGET_DATADIR:|$(DATADIR)|" \
@@ -52,7 +49,7 @@ chcon:
 daemon-reload:
 	-systemctl daemon-reload
 
-install: base units useradd chcon daemon-reload
+install: base units
 
 uninstall-base:
 	rm -fv $(DESTDIR)$(SBINDIR)/memavaild
-- 
2.26.2

